build/azDevOps/azure/azure-pipelines-netcore-k8s.yml

```diff
@@ -197,6 +203,38 @@ stages:
               functional_test_artefact: tests
               repo_name: $(self_repo)
 
+          # Copy the files into the correct place for packaging
+          - task: Bash@3
+            displayName: Prepare for Packaging
+            inputs:
+              targetType: "inline"
+              workingDirectory: $(Agent.BuildDirectory)/s
+              script: |
+                cp $REPO_NAME/template.csproj .
+                mkdir ./templates
+                pushd templates
+                cp -r ${ROOT_SRC_DIR}/$REPO_NAME .
+                cp -r ${ROOT_SRC_DIR}/$REPO_NAME/src/api .
```

Comment:
You don't need the /api folder on this one.
This repo contains a single template, which is the full api

Reply:
OK, have removed this